src/Mod/AddonManager/addonmanager_utilities.py

```diff
@@ -296,21 +286,50 @@ def getDescRegex(url):
     """Returns a regex string that extracts a WB description to be displayed in the description
     panel of the Addon manager, if the README could not be found"""
 
-    if ("github" in url):
+    if "github" in url:
         return "<meta property=\"og:description\" content=\"(.*?)\""
-    elif ("framagit" in url):
+    elif "framagit" in url or "gitlab" in url:
         return "<meta.*?content=\"(.*?)\".*?og\:description.*?>"
     print("Debug: addonmanager_utilities.getDescRegex: Unknown git host:",url)
     return None
 
-def getRepoUrl(text):
+
+def getReadmeHTMLUrl(url):
```

Comment:
If you are adding new functions please use proper Pythonic style, meaning `snake_case` functions. FreeCAD is not entirely PEP8 compliant but we strive for it little by little.